backend/src/main/kotlin/cz/loono/backend/db/model/HealthcareProvider.kt

```diff
@@ -113,7 +114,7 @@ data class HealthcareProvider(
     @Column(nullable = false, columnDefinition = "TEXT")
     val specialization: String? = null,
 
-    @ManyToMany(cascade = [CascadeType.ALL])
+    @ManyToMany(cascade = [CascadeType.ALL], fetch = FetchType.EAGER)
```

Comment:
Prečo eager? Zvyšuje to spotrebu RAM. Znižuje počet dotazov do DB.

Reply:
Rozdelil jsem transakce a nyni nejsou v healthcare providers natazene categories. S late init to vyhodi vyjimku.